index.d.ts

```diff
@@ -91,7 +91,7 @@ declare namespace Router {
     ): void;
 
     constraints? : {
-      [key: string]: ConstraintStrategy<V>
+      [key: string]: ConstraintStrategy<V, unknown>
```

Comment:
This seems like it might be a breaking change, do the fastify type tests still pass with this in place?